src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -2,22 +2,22 @@
 
 import static java.util.Objects.requireNonNull;
 
-import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
+import seedu.address.model.PersonBook;
 
 /**
  * Clears the address book.
  */
 public class ClearCommand extends Command {
```

Comment:
So the Clear Command is only for clearing the Personslist?

Reply:
Good point, changed to clear both property and person lists!